### PR TITLE
force omission of meta files

### PR DESCRIPTION
**Pull request by ryanfkeepers**
While the `isMeta` flag will solve this in the
future, the current state requires us to filter
out all meta and dirmeta files, even historically.

---

#### Does this PR need a docs update or release note?

- [x] :white_check_mark: Yes, it's included

#### Type of change

- [x] :bug: Bugfix

#### Issue(s)

* #1970

#### Test Plan

- [x] :zap: Unit test
- [x] :green_heart: E2E

### CODE REVIEW

**Review by zackrossman — APPROVED** [review]
LGTM, FYI @rajatkateja I know you've been working in/around this issue

File: src/pkg/backup/details/details.go [hunk 1]
```diff
@@ -127,7 +128,13 @@ func (dm DetailsModel) FilterMetaFiles() DetailsModel {
...
 // be treated as regular files.
 func (de DetailsEntry) isMetaFile() bool {
-	return de.ItemInfo.OneDrive != nil && de.ItemInfo.OneDrive.IsMeta
+	if de.ItemInfo.OneDrive == nil {
+		return false
+	}
+
+	return de.ItemInfo.OneDrive.IsMeta ||
+		strings.HasSuffix(de.RepoRef, ".meta") ||
+		strings.HasSuffix(de.RepoRef, ".dirmeta")
```
Comment (ashmrtn) [inline src/pkg/backup/details/details.go:137]: nit should use OneDrive consts if possible

**Review by ashmrtn — APPROVED** [review]

File: src/pkg/backup/details/details.go (hunk 1 above)
Comment (ryanfkeepers) [reply src/pkg/backup/details/details.go:137]: Nevermind!  Circular imports there.

Comment (ashmrtn) [reply src/pkg/backup/details/details.go:137]: hmmm alright. Thanks for testing it out. I think maybe it will be cleaned up in a later patch

**Pull request merged**